frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -357,53 +406,115 @@ const PropertyListView: React.FC = () => {
               <FaFileAlt data-testid="csv-icon" size={36} onClick={() => fetch('csv')} />
             </FileIcon>
           </TooltipWrapper>
+          <VerticalDivider />
+
+          <TooltipWrapper toolTipId="edit-financial-values" toolTip={'Edit financial values'}>
+            <EditIconButton>
+              <FaEdit data-testid="edit-icon" size={36} onClick={() => setEditable(!editable)} />
+            </EditIconButton>
+          </TooltipWrapper>
+          {editable && (
+            <TooltipWrapper
+              toolTipId="save-edited-financial-values"
+              toolTip={'Save financial values'}
+            >
+              <Button
+                data-testid="save-changes"
+                disabled={dirtyRows.length === 0}
+                onClick={() => {
+                  if (tableFormRef.current?.dirty) {
+                    tableFormRef.current.submitForm();
+                  }
+                }}
+              >
+                Save edits
+              </Button>
+            </TooltipWrapper>
+          )}
         </Container>
-        <Table<IProperty>
-          name="propertiesTable"
-          lockPageSize={true}
-          columns={columns}
-          data={data || []}
-          loading={data === undefined}
-          filterable
-          sort={sorting}
-          pageIndex={pageIndex}
-          onRequestData={handleRequestData}
-          pageCount={pageCount}
-          onSortChange={(column: string, direction: SortDirection) => {
-            if (!!direction) {
-              setSorting({ ...sorting, [column]: direction });
-            } else {
-              const data: any = { ...sorting };
-              delete data[column];
-              setSorting(data);
-            }
-          }}
-          canRowExpand={(val: any) => {
-            if (val.values.propertyTypeId === 0) {
-              return true;
-            } else {
-              return false;
-            }
-          }}
-          detailsPanel={{
-            render: val => {
-              if (expandData[val.id]) {
-                return <Buildings hideHeaders={true} data={expandData[val.id]} />;
+        <Formik
+          innerRef={tableFormRef as any}
+          initialValues={{ properties: data }}
+          enableReinitialize
+          onSubmit={async values => {
+            let nextProperties = [...values.properties!];
```

Comment:
I'm a bit worried about these !

Reply:
Nice catch, was using this to bypass an error while testing